templates/content/field--node--oe-sc-event-registration-url.html.twig

```diff
@@ -0,0 +1,21 @@
+{#
+/**
+ * @file
+ * Event registration button display.
+ */
+#}
+<div class="mt-4">
+  {% set button %}
+    {{ pattern('button', {
+      'label': 'Register'|t,
+      'icon': 'calendar-check',
+      'icon_position': 'before',
+    }) }}
+  {% endset %}
+  {% set _attributes = element[0]['#options']['attributes'] %}
+  {{ pattern('link', {
+    'label': button,
+    'path': element[0]['#url'].toString(),
+    'attributes': _attributes,
+  }) }}
+</div>
```

Comment:
Note:
I notice that in netlify the button classes are directly on the link tag.
I find this in `src/compositions/bcl-event/data/data-page.js`:
```
    links: [
      {
        label: "Register",
        icon_position: "before",
        icon: {
          name: "calendar-check",
          path: "/icons.svg",
        },
        attributes: new drupalAttribute().addClass([
          "btn",
          "btn-primary",
          "d-block",
          "d-md-inline-block",
        ]),
      },
    ],
```
So their solution would require to pass explicit classes to the content banner pattern or bcl component, which we want to avoid. So ok for this solution.

Reply:
Even better would be a bcl component or pattern for "button link". But not today.